.travis.yml

```diff
@@ -0,0 +1,46 @@
+sudo: false
```

Comment:
Hm, all the small little differences add up. We could do it manually, but distributing sounds like a benefit. I think the .travis file will end up being 100% the same for all projects. The only thing that Fireplace does differently is set up Flue, and other projects are going to need Flue as they get more test coverage.